<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>punishmentDialog</class>
 <widget class="QDialog" name="punishmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uname_edt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查  询</string>
   </property>
  </widget>
  <widget class="QTableView" name="res_tbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="done_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>处  理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rtn_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返  回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
